views/wsprov_fetch_linked_objects.aql

```diff
@@ -4,12 +4,21 @@
 // Args:
 //   ws_ids - array of private workspace ids the user has access to
 //   obj_keys - wsprov_object key to find links for
+//   is_private - if present, limit to objects in workspaces that a user has access to
+//   is_public - if present, limit to objects in public workspaces
+//   *** if both is_private and is_public are true this will be treated as an OR ***
```

Comment:
"show_public" and "show_private" might better convey the meaning of these variables

Reply:
I agree